storage/core/src/server/Interfaces.ts

```diff
@@ -17,9 +17,21 @@ export type TransferType = "buffer" | "stream" | "local";
 export type TransferData = Readable | Buffer | string;
 export type MultipartUploadData = Readable | string;
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AbortSignalListener = (this: AbortSignal, ev: any) => any;
+
+export interface AbortSignal {
```

Comment:
This seems like it could cause issues, because it's using the same name as a global object.